app/config/settings_prod.py

```diff
@@ -148,6 +148,7 @@
 STATIC_ROOT = os.path.join(BASE_DIR, 'var/www/stac_api/static_files')
 STATICFILES_DIRS = [
     BASE_DIR / "spec/static",
+    BASE_DIR / "app/stac_api/templates"
```

Comment:
I think the templates itself don't need to be in staticfiles. If it's just for the `*.js` file, pls move the `*.js` file to `stac_api/static`, then it should automatically be found by Django and you can remove this entry.

Reply:
ok, I thought that we need this that manager.py collectstatic works